datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py

```diff
@@ -62,16 +65,21 @@ def clean(ctx, check, compiled_only, all_matches, force, verbose):
         if basepath(path) in ('integrations-core', 'integrations-extras'):
             if force:
                 force_clean_root = True
-            else:
+            elif not compiled_only:
                 echo_warning(
-                    'You are running this from the root of the integrations project. '
-                    'Should we remove everything, including: '
-                    '.cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info? '
-                    'You can also use --force or -f to bypass this input.'
+                    'You are running this from the root of the integrations project.\n'
+                    'By default, the following artifacts in the root directory will *not* be cleaned:'
+                )
+                echo_info('.cache, .coverage, .eggs, .pytest_cache, .tox, build, dist, and *.egg-info')
```

Comment:
(nit) We should refer to this list, instead of hard coding: https://github.com/DataDog/integrations-core/blob/8f7b8b0d7757fb14858d8c638c037e8047d2e71e/datadog_checks_dev/datadog_checks/dev/tooling/clean.py#L12

extra: can also be used for the command desc: https://github.com/DataDog/integrations-core/blob/8f7b8b0d7757fb14858d8c638c037e8047d2e71e/datadog_checks_dev/datadog_checks/dev/tooling/commands/clean.py#L47-L49